runtime/ftplugin/zathurarc.vim

```diff
@@ -0,0 +1,22 @@
+" Vim filetype plugin file
+" Language:             Zathurarc
+" Maintainer:           Wu, Zhenyu <wuzhenyu@ustc.edu>
+" URL:                  https://pwmt.org/projects/zathura/documentation/
+" Latest Revision:      2024-04-02
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:save_cpoptions = &cpoptions
+set cpoptions&vim
+
+let b:undo_ftplugin = 'setlocal comments< commentstring< include< iskeyword<'
+setlocal comments=:#
+setlocal commentstring=#\ %s
+setlocal include=^\sinclude
+setlocal iskeyword+=-
```

Comment:
are you setting this because of the syntax keywords containing a `-`? In that case please use the `:syn iskeyword` command instead

Reply:
Fixed.